zeppelin-web/README.md

```diff
@@ -19,19 +19,27 @@ It is recommended to install node 6.0.0+ since Zeppelin uses 6.9.1+ (see [creati
 
 All build commands are described in [package.json](./package.json)
 
-```sh
 # install required depepdencies and bower packages (only once)
-$ npm install -g yarn
-$ yarn install
+```
+$ npm install -g yarn bower npm-run-all rimraf grunt webpack karma karma-phantomjs-launcher
```

Comment:
@DmytroShkvyra I see. Which environment did you use when it requires install 'rimraf grunt webpack karma karma-phantomjs-launcher' globally? 

Because on linux, osx, `mvn package` success without installing them globally.